web/impact/impact/tests/test_organization_history_view.py

```diff
@@ -333,6 +334,45 @@ def test_startup_became_finalist_no_created_at(self):
             self.assertEqual(1, len(events))
             self.assertEqual(start_date, events[0]["datetime"])
 
+    def test_startup_became_winner(self):
+        startup = StartupFactory()
+        startup_status = StartupStatusFactory(
+            program_startup_status__startup_role__name=StartupRole.SILVER_WINNER,
+            startup=startup)
+        program = startup_status.program_startup_status.program
+        role = startup_status.program_startup_status.startup_role
+        with self.login(email=self.basic_user().email):
+            url = reverse(OrganizationHistoryView.view_name,
+                          args=[startup.organization.id])
+            response = self.client.get(url)
+            events = find_events(response.data["results"],
+                                 OrganizationBecameWinnerEvent.EVENT_TYPE)
+            self.assertEqual(1, len(events))
```

Comment:
This test seems to be making about half a dozen assertions. We generally prefer to test one thing at a time. What exactly are we trying to test here?

Reply:
See previous comment.